kconfigs/fvp.conf

```diff
@@ -5,3 +5,4 @@ CONFIG_TCG_FTPM_TEE=m
 CONFIG_9P_FS=y
 CONFIG_NET_9P=y
 CONFIG_NET_9P_VIRTIO=y
+CONFIG_ARM_FFA_TRANSPORT=y
```

Comment:
In general it might be cleaner to use `./scripts/config --(dis|en)able` for options which depend on some condition. (Same applies to my previous comment in this file assuming your changes depend on conditions.)

Reply:
AFAICS the convention for other platforms is to enable everything that might be needed. E.g. for QEMU the Virtio 9p, FF-A driver, etc are always enabled, regardless of the build config. I think the benefit of this is that when changing the build config we don't have to (partially) rebuild the kernel, which might take a long time.

Reply:
Ok, thanks! leave it as is then.